projects/services/public-api-service-typescript/test/public-api-service.spec.ts

```diff
@@ -17,6 +17,20 @@ const contractAddressBase58 = "T00LUPVrDh4SDHggRBJHpT8hiBb6FEf2rMkGvQPR";
 const transactionTimestamp = "12345678";
 const payload = JSON.stringify({ foo: "bar" });
 
+const transaction: types.Transaction = {
```

Comment:
why not use a transaction builder? this test should not be intimately familiar with how addresses in a transaction are encoded

Reply:
IMO it should, because what it tests is that an http request (that has a b58-encoded address) is decoded well before propagated to the TX pool